conference.js

```diff
@@ -2091,8 +2091,9 @@ export default {
         room.on(JitsiConferenceEvents.USER_ROLE_CHANGED, (id, role) => {
             if (this.isLocalId(id)) {
                 logger.info(`My role changed, new role: ${role}`);
-
+                
```

Comment:
Change your IDE to not use tabs and to remove unneeded whitespaces. Please remove this one, thanks.